packages/docusaurus-theme-search-algolia/package.json

```diff
@@ -33,7 +33,7 @@
     "copy:watch": "node ../../admin/scripts/copyUntypedFiles.js --watch"
   },
   "dependencies": {
-    "@docsearch/react": "^3.2.1",
+    "@docsearch/react": "^3.3.0",
```

Comment:
Apparently, there are type-related breaking changes in this new version @shortcuts @Haroenv

This upgrade leads to failures while type-checking our website: https://github.com/facebook/docusaurus/actions/runs/3343786713/jobs/5537374683

```
Run yarn workspace website typecheck
yarn workspace v1.22.19
yarn run v1.22.19
$ tsc
Error: ../node_modules/@algolia/autocomplete-preset-algolia/dist/esm/types/algoliasearch.d.ts(7,7[4](https://github.com/facebook/docusaurus/actions/runs/3343786713/jobs/5537374683#step:7:5)): error TS2339: Property 'liteClient' does not exist on type 'typeof import("/home/runner/work/docusaurus/docusaurus/node_modules/algoliasearch/lite")'.
Error: ../node_modules/@algolia/autocomplete-preset-algolia/dist/esm/types/algoliasearch.d.ts(8,7[5](https://github.com/facebook/docusaurus/actions/runs/3343786713/jobs/5537374683#step:7:6)): error TS2339: Property 'searchClient' does not exist on type 'typeof import("/home/runner/work/docusaurus/docusaurus/node_modules/@algolia/client-search/dist/client-search")'.
Error: ../node_modules/@algolia/autocomplete-preset-algolia/dist/esm/types/algoliasearch.d.ts(20,23): error TS2[6](https://github.com/facebook/docusaurus/actions/runs/3343786713/jobs/5537374683#step:7:7)94: Namespace '"/home/runner/work/docusaurus/docusaurus/node_modules/algoliasearch/lite"' has no exported member 'LegacySearchMethodProps'.
Error: ../node_modules/@algolia/autocomplete-preset-algolia/dist/esm/types/algoliasearch.d.ts(24,23): error TS2694: Namespace '"/home/runner/work/docusaurus/docusaurus/node_modules/algoliasearch/lite"' has no exported member 'SearchForFacetValuesResponse'.
Error: ../node_modules/@algolia/autocomplete-preset-algolia/dist/esm/types/algoliasearch.d.ts(2[8](https://github.com/facebook/docusaurus/actions/runs/3343786713/jobs/5537374683#step:7:9),23): error TS2694: Namespace '"/home/runner/work/docusaurus/docusaurus/node_modules/algoliasearch/lite"' has no exported member 'SearchResponse'.
Error: ../node_modules/@algolia/autocomplete-preset-algolia/dist/esm/types/algoliasearch.d.ts(32,23): error TS26[9](https://github.com/facebook/docusaurus/actions/runs/3343786713/jobs/5537374683#step:7:10)4: Namespace '"/home/runner/work/docusaurus/docusaurus/node_modules/algoliasearch/lite"' has no exported member 'HighlightResult'.
Error: ../node_modules/@algolia/autocomplete-preset-algolia/dist/esm/types/algoliasearch.d.ts(36,23): error TS2694: Namespace '"/home/runner/work/docusaurus/docusaurus/node_modules/algoliasearch/lite"' has no exported member 'SnippetResult'.
```

Seems related to https://github.com/algolia/autocomplete/pull/1002

Reply:
Thanks for letting me know, the ts-ignore comments were removed :( Will fix!